pkg/ingester/ingester.go

```diff
@@ -109,6 +96,13 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.MaxChunkAge, "ingester.max-chunk-age", 12*time.Hour, "Maximum chunk age before flushing.")
 	f.IntVar(&cfg.ConcurrentFlushes, "ingester.concurrent-flushes", DefaultConcurrentFlush, "Number of concurrent goroutines flushing to dynamodb.")
 	f.StringVar(&cfg.ChunkEncoding, "ingester.chunk-encoding", "1", "Encoding version to use for chunks.")
+
+	// DEPRECATED, no-op
+	f.Bool("ingester.reject-old-samples", false, "DEPRECATED. Reject old samples.")
```

Comment:
should probably say something in the PR description about upgrade procedure - need to enable the checks in the new distributor before upgrading the ingester

Reply:
Done.